.travis.yml

```diff
@@ -7,10 +9,27 @@ php:
   - 7.0
   - 7.1
   - nightly
-  - hhvm
```

Comment:
as far as I see, if this stays here, there will be tests run against the old version and the new version so we could keep this to test against two versions of hhvm.